hoverctl/cmd/mode.go

```diff
@@ -24,14 +29,37 @@ mode is shown.
 
 			fmt.Println("Hoverfly is currently set to", mode, "mode")
 		} else {
-			mode, err := hoverfly.SetMode(args[0])
+			modeView := v2.ModeView{
+				Mode: args[0],
+			}
+
+			var headersMessage string
+			if allHeaders {
+				modeView.Arguments.Headers = append(modeView.Arguments.Headers, "*")
+
+				headersMessage = "and will capture all request headers"
+			} else if len(specficHeaders) > 0 {
+				modeView.Arguments.Headers = append(modeView.Arguments.Headers, strings.Split(specficHeaders, ",")...)
+
+				if strings.Contains(specficHeaders, ",") {
```

Comment:
Maybe removing the if else statement, and just print one message with 'request header(s')?